engines/advancedDetector.cpp

```diff
@@ -325,17 +325,27 @@ Common::Error AdvancedMetaEngine::createInstance(OSystem *syst, Engine **engine)
 		return Common::kNoError;
 }
 
-void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFilePropertiesMap &filesProps) const {
-	// TODO: This message should be cleaned up / made more specific.
-	// For example, we should specify at least which engine triggered this.
-	//
-	// Might also be helpful to display the full path (for when this is used
-	// from the mass detector).
+void AdvancedMetaEngine::reportUnknown(const Common::FSNode &path, const ADFilePropertiesMap &filesProps, const ADGameIdList &matchedGameIds) const {
 	Common::String report = Common::String::format(
-			_("The game in '%s' seems to be unknown.\n"
-			  "Please, report the following data to the ScummVM team along with name\n"
-			  "of the game you tried to add and its version, language, etc.:"), path.getPath().c_str()) + "\n";
-	report += "\n";
+			_("The game in '%s' seems to be an unknown %s engine game variant.\n\n"
+			  "Please report the following data to the ScummVM team at "
+			  "https://bugs.scummvm.org along with the name of the game "
+			  "you tried to add and its version, language, etc.:"), path.getPath().c_str(), getName(), "https://bugs.scummvm.org/");
+
```

Comment:
It looks like you half intended to use a %s for the bug tracker path?

Also it seem strange to translate this part of the message, but not the part added to it below (the `"Matched game IDs:"` bit).